client/public/js/components/EventList.js

```diff
@@ -30,7 +27,8 @@ var EventList = React.createClass({
   getInitialState: function() {
     // Set initial state to the initial set of events defined in EventStore
     return {
-      events: getEvents(),
+      shepherdEvents: EventStore.getAllEventsByShepherd(cookie.load('id')),
+      notShepherdEvents: EventStore.getAllEventsNotByShepherd(cookie.load('id')),
```

Comment:
`getAllEventsByShepherd` and `getAllEventsNotByShepherd` on the EventStore do not take any arguments.